rasa/nlu/components.py

```diff
@@ -87,16 +87,43 @@ def validate_required_components_from_data(
     pipeline: List["Component"], data: TrainingData
 ):
 
-    response_selector_exists = False
-    for component in pipeline:
-        # check if a response selector is part of NLU pipeline
-        if MESSAGE_RESPONSE_ATTRIBUTE in component.provides:
-            response_selector_exists = True
+    if data.regex_features and not any(
+        [component.name == "RegexFeaturizer" for component in pipeline]
+    ):
+        warnings.warn(
+            "You have defined training data with regexes, but "
+            "your NLU pipeline does not include a RegexFeaturizer. "
+            "To featurize regexes, add a RegexFeaturizer to your pipeline."
```

Comment:
This should be the second sentence for the lookups and regexes too :)

Reply:
got that change implemented